rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+
+The imposition of specific requirements to metadata is part of this RFC proposal and a key component of the proposed DAPS solution.
+
+This approach provides a generalized signaling mechanism suitable for all foreseeable data types while accounting for data type-specific requirements.
+
+The DAPS approach has common elements with project-level bundles that have been proposed in other contexts, but differs in the following ways:
+
+1. It does not impose any specific aggregation level on the data allowing for more flexibility and accommodating future unforeseen processing scenarios
+2. Allows specification of bundles included in a DAPS in many alternative different ways (listing, search, etc…)
+3. Does not impose the creation of a new bundle type that is tied to a specific data modality
+4. Dictates specific metadata requirements that guarantee the correct processing of all foreseeable data modalities
+5. Decouples analysis initiation from data representation allowing for future flexibility
+
+## General Metadata Requirements
+In the above framework, analysis pipelines will receive a set of bundles that _potentially_ need to be co-processed. As aforementioned, this signal will not explicitly identify the bundles that need to be co-processed in a single pipeline, instead, it will serve as a notification **indicating that any co-processing is to be done only within the boundaries of this set of files (submission envelope). **
+
+Analysis can initiate multiple pipeline invocations as a result of a single DAPS being submitted.
+
+The actual grouping of the files is to be performed by analysis via a search in the metadata of the bundles submitted following well-defined modality-specific set of rules. For this reason, the metadata must satisfy the following metadata requirements (MR):
+
+Furthermore, if part of the data submitted in a DAPS has already been processed the Analysis infrastructure will only re-process the parts that have not been modified.
+
+1. The data modality of each data file included in the DAPS must be clearly identifiable (MR1)
+2. Data that need to be processed together must be clearly identifiable by a modality-specific set of metadata variables that will group the files according to processing requirements. (MR2)
+3. Among the data that needs to be processed together, the metadata must provide sufficient information to correctly order, pair or otherwise establish correspondence and type of the input files. (MR3)
+
+## Proposed Signalling Implementation
+The DAPS proposal above only sets specific requirements for the signal to initiate data analysis and minimum requirements for the associated metadata. We hereby provide a specific recommendation for implementation via bundle notifications.
```

Comment:
What is the purpose of inventing a new type of aggregate structure in the DSS. Why not create another bundle type? - this seems to be adding complexity for no clear advantage

Reply:
DAPS is a bundle type that is used to group other bundles.  If this isn't clear, we need to update the text and for sure add an image.

Reply:
That isn't clear at all. Some diagrams would be a great help here